pkg/plugins/resources/k8s/store_template_test.go

```diff
@@ -23,8 +19,6 @@ var _ = Describe("KubernetesStore template", func() {
 		Expect(kubeTypes.RegisterObjectType(&sample_proto.TrafficRoute{}, &sample_k8s.SampleTrafficRoute{})).To(Succeed())
 		Expect(kubeTypes.RegisterListType(&sample_proto.TrafficRoute{}, &sample_k8s.SampleTrafficRouteList{})).To(Succeed())
 
-		k8sClient.Create(context.Background(), &kube_core.Namespace{ObjectMeta: kube_meta.ObjectMeta{Name: "demo"}})
```

Comment:
I think this is needed. In a store template we use `resource1.demo` for names which on K8S means resource of name `resource1` with namespace `demo`

Reply:
This was moved to the BeforeSuite, because we need it created only once, and this is getting called several times when the creation is here.

Reply:
I missed that, thanks!